src/USER-MISC/README

```diff
@@ -65,7 +65,6 @@ pair_style buck/mdf, Paolo Raiteri, p.raiteri at curtin.edu.au, 2 Dec 15
 pair_style coul/diel, Axel Kohlmeyer, akohlmey at gmail.com, 1 Dec 11
 pair_style dipole/sf, Mario Orsi, orsimario at gmail.com, 8 Aug 11
 pair_style edip, Luca Ferraro, luca.ferraro at caspur.it, 15 Sep 11
-pair_style eam/cd, Alexander Stukowski, stukowski at mm.tu-darmstadt.de, 7 Nov 09
```

Comment:
Shouldn't the author be mentioned in the eam package now?

Reply:
the MANYBODY package doesn't have a README file. this is a special case for USER-MISC. The author (and the corresponding publication) **is** already mentioned in the EAM pair style docs.